crates/aptos-telemetry-service/src/prometheus_push_metrics.rs

```diff
@@ -45,6 +45,10 @@ pub async fn handle_metrics_ingest(
 ) -> anyhow::Result<impl Reply, Rejection> {
     debug!("handling prometheus metrics ingest");
 
+    let enable_location_labels = env::var("FEATURE_LOCATION_LABELS_ENABLED")
+        .map(|val| val.parse::<bool>().unwrap_or(false))
+        .unwrap_or(false);
+
     let enable_random_label = env::var("FEATURE_RANDOM_LABEL_ENABLED")
```

Comment:
Did we decide to keep this feature?

Reply:
I'm planning to keep this disabled by default. I still haven't figured out the issue where the labels alternate between location and no location

Reply:
I meant the `FEATURE_RANDOM_LABEL_ENABLED`?

Reply:
Oh, I think this still might be useful to keep around for loadtesting our selfhosted VM instance